doc/userguide/lua/libs/ssh.rst

```diff
@@ -95,3 +100,56 @@ Example::
   local tx = ssh.get_tx()
   local software = tx:client_software();
   print (software)
+
+``client_hassh()``
```

Comment:
@jasonish should we have a `hassh(dir)` lua function that takes a parameter ?

Reply:
I don't why.